code/script.js

```diff
@@ -2,6 +2,10 @@
 const board = document.getElementById('board')
```

Comment:
In this area some of the DOM's have semicolons. Just a heads up to double check and choose if you want to go with semicolons or not. I actually did the same in my code so I will also need to double check my own :)

Reply:
You need to check your VScode settings and search for semicolons and if you decide to have them then make Prettier(vscode's extension- if you don't have it you need to install it) add them automatically after each saves.